### ISSUE TITLE
Series Highlighting in LineCharts

### ISSUE THREAD

**Issue by jzillmann** [recharts/recharts#909]
For line charts, when you hover through the chart, all points on the x-axis data point are highlighted / tool tipped.

Now is there a way to highlight/tooltip a individual line/series instead ?
Like done here: http://dygraphs.com/gallery/#g/highlighted-series ?

**Comment by kamthamc** [comment]
@jzillmann i think you are creating issue in wrong repository. https://github.com/danvk/dygraphs

**Comment by jzillmann** [comment]
@kamthamc No, the dygraphs example just shows what i want to achieve with recharts!

**Comment by kamthamc** [comment]
@jzillmann sorry, i misunderstood the question.

**Comment by jzillmann** [comment]
np!

**Comment by mrfy** [comment]
Hi, is there any solution for this? I see that topic is closed but cannot find anything. Thank you

**Comment by ckifer** [comment]
Re-opened labeled and adjusted name a bit

**Comment by girirajSpurFit** [comment]
is this possible in recharts now
